launch-web-service.sh

```diff
@@ -1,3 +1,17 @@
 #!/usr/bin/env bash
+HOST_IP=`ip addr show | egrep "e(th|np)" | grep inet | awk '{print $2}' | sed -e 's/\/[0-9]*//'`
```

Comment:
Use $(...) notation instead of legacy backticked `...`. (SC2006)

Reply:
Implemented the suggested code change.